gointelowl/gointelowl.go

```diff
@@ -0,0 +1,97 @@
+package gointelowl
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+// Struct for IntelOwl.
+// Initiate it using Token, Instance, Certificate
+type IntelOwl struct {
+	Token       string
+	Instance    string
+	Certificate string
+}
+
+// Fetch list of all tags.
+//        Endpoint: ``/api/tags``
+// It Returns Slice of Tags
+func (intelOwl IntelOwl) GetAllTags() []map[string]string {
```

Comment:
this looks little confusing to me, can you use `ioClient IntelOwlClient` as the variable name instead ?